docs/Features/XMLFeatures.md

```diff
@@ -0,0 +1,139 @@
+# XML Features
+
+<!--  -->
+
+## XML Tag Auto Close
+
+[vscode-xml](https://github.com/redhat-developer/vscode-xml) provides an `auto close` feature for XML tags:
```

Comment:
Please remove vscode-xml ....Redo the sentence like XML Tag auto close is supported.